apps/full-site-editing/full-site-editing-plugin/donations/stripe-nudge.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+/**
+ * WordPress dependencies
+ */
+// eslint-disable-next-line no-restricted-imports
+import { Button, Dashicon } from '@wordpress/components';
+import { Warning } from '@wordpress/block-editor';
+import { compose } from '@wordpress/compose';
+import { __ } from '@wordpress/i18n';
+import { withDispatch } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+
+const StripeNudge = ( { autosaveAndRedirect, stripeConnectUrl } ) => (
+	<Warning
+		actions={
+			// Use href to determine whether or not to display the Upgrade button.
+			stripeConnectUrl && [
+				<Button
+					key="connect"
+					href={ stripeConnectUrl } // Only for server-side rendering, since onClick doesn't work there.
+					onClick={ autosaveAndRedirect }
+					target="_top"
+					isDefault
+					className="premium-content-block-nudge__button stripe-nudge__button"
```

Comment:
Do we intend to reuse the premium content classes here? If so, we need to load the [premium content editor styles](https://github.com/Automattic/wp-calypso/blob/master/apps/full-site-editing/full-site-editing-plugin/premium-content/editor.css).

Reply:
They were already being enqueued, but that wasn't really good either 🙃  I'll do a PR.